packs/data/spell-effects.db/spell-effect-wish-twisted-form-acid.json

```diff
@@ -0,0 +1,55 @@
+{
+    "_id": "2ECXDf14cKs3fQzO",
+    "img": "systems/pf2e/icons/spells/wish-twisted-form.webp",
+    "name": "Spell Effect: Wish-Twisted Form (Acid)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Wish-Twisted Form]{Wish-Twisted Form}</em>.</p>\n<p>Implemented Effects:</p>\n<ul>\n<li>The target gains weakness 5 to acid damage.</li>\n<li>The target takes a -10-foot status penalty to all its Speeds.</li>\n<li>The target takes a -1 status penalty to its saving throws.</li>\n</ul>\n<p>Unimplemented Effects:</p>\n<ul>\n<li>The target's resistances are reduced by 5.</li>\n</ul>\n<hr />\n<p><strong>Heightened (+1)</strong> Increase the weakness and the reduction to resistances by 1.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 5
+        },
+        "rules": [
+            {
+                "key": "Weakness",
+                "type": "acid",
```

Comment:
Could these be a single effect with a ChoiceSet for the weakness type?

Reply:
Thanks for the suggestion, that does sound a lot better. Just implemented that.